styles.css

```diff
@@ -439,6 +439,84 @@ button {
   color: #fff;
 }
 
+.contact-section {
+  padding: 20% 4% 0%;
+}
+
+.contact-section h2 {
+  font-size: 2em;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  text-align: center;
+  color: #172b4d;
+  padding: 0;
+  margin: 0;
+}
+
+.contact-section ul {
+  padding: 0;
+}
+
+.contact-section li {
+  display: flex;
+  flex-direction: column;
+  align-self: center;
+}
```

Comment:
Good job trying to implement `Flexbox` in your form. 🔥 However, your current flex property does not have any effect on the form because it is assigned to the child element. 

- [ ] I would suggest that you use the `display: flex` property on the parent element `<ul>` instead of the `<li>`. This was the property his properly used. 
You would rather adjust the width of the child items so it fits the page as in the Figma design. 🎨

Reply:
Excellent feedback. I really appreciate your time 😄